src/components/Restoration/YourCompanySummary.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card flat id="restoration-summary">
+    <!-- Section Header -->
+    <div class="summary-header px-4 mb-2 rounded-t">
+      <v-row no-gutters>
+        <v-col cols="9">
+          <img class="header-icon" src="@/assets/images/currency-usd-circle.svg">
+          <label class="summary-title">Your Company</label>
+        </v-col>
+      </v-row>
+    </div>
+    <!-- Business Name -->
+    <template>
+      <v-divider class="mx-4" />
+      <div class="section-container business-name-summary">
+        <v-row no-gutters>
+          <v-col cols="3">
+            <label><strong>Restoration Type</strong></label>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedExtendRestorationFiling">
+            <div class="font-weight-bold">Limited Restoration Extension</div>
+            <div>Extension Time: {{ getFormattedExpiryText }}</div>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedConversionRestorationFiling">
+            <div class="font-weight-bold">Conversion to Full Restoration</div>
+            <div>[TODO - Applicant's relationship: Director, Shareholder]</div>
+          </v-col>
+        </v-row>
+        <v-row no-gutters class="mt-3" v-if="hasFileNumber">
+          <v-col cols="3">
+            <label><strong>Approval Type</strong></label>
+          </v-col>
+          <v-col cols="8" class="mt-n1">
+            <div class="font-weight-bold">Approved by Court Order</div>
+            <div>Court Order Number: {{ getFileNumber }}</div>
+          </v-col>
+        </v-row>
+      </div>
+    </template>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { mapGetters } from 'vuex'
+
+export default {
+  computed: {
+    ...mapGetters(['isLimitedConversionRestorationFiling', 'isLimitedExtendRestorationFiling',
+      'hasFileNumber', 'getFileNumber', 'getFormattedExpiryText'])
```

Comment:
I know you like these, but we lose typing this way. Thoughts?

Reply:
Oh, right, I meant to call that out. The `@Getter` syntax is only available if we're using the Class API (thanks for providing a name to that pattern).  Since the Class API isn't available in Vue3 and we have upgrade prior to December 2023, I thought I'd use the Options API for any new components.  What do you think?

Reply:
I don't know, so let's just leave this.